content/en/docs/reference/command_line_interface/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "CLI - Command line interface"
```

Comment:
Similarly, this directory can be named as 'cli'. Everyone knows what `cli` means.

Reply:
can we keep kubectl out of this PR or untouched?